src/platform/site-wide/user-nav/components/SearchMenu.jsx

```diff
@@ -225,7 +225,8 @@ export class SearchMenu extends React.Component {
             />
             <button
               type="submit"
-              disabled={!isUserInputValid}
+              id="sitewide-search-submit-button"
```

Comment:
Since this is for the e2e tests, could you use `data-testid` or `data-e2e-id` or something similar?

Reply:
Good call!